### PR TITLE
(SIMP-2249) Create FIPS module

### PR DESCRIPTION
**Pull request by jeefberkey**
Moved FIPS content from simplib to a new module.

SIMP-2249 #close
SIMP-1679 #comment Created FIPS module

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -1,143 +1,52 @@
...
-  Class[ '::fips::config'  ] ~>
-  Class[ '::fips::service' ] ->
-  Class[ '::fips' ]
-
-  if $enable_auditing {
-    include '::fips::config::auditing'
-    Class[ '::fips::config::auditing' ] ->
-    Class[ '::fips::service' ]
+  Boolean $enabled = true,
+  Boolean $aesni   = $::cpuinfo and member($::cpuinfo['processor0']['flags'],'aes')
```
Comment (trevor-vaughan) [inline manifests/init.pp]: This should probably be getvar() in case it doesn't exist and strict_variables is on

Comment (jeefberkey) [inline manifests/init.pp]: It should probably go in params.pp then

**Comment by jeefberkey** [comment]
@nick-markowski Looks like that was the templated acceptance test from the skeleton, it will be deleted.

**Comment by jeefberkey** [comment]
Fixing the other broken acceptance tests.

**Pull request merged**